proprietary/tokens/src/components/ams/accordion.tokens.json

```diff
@@ -5,11 +5,11 @@
       "button": {
         "color": { "value": "{ams.color.interactive.default}" },
         "cursor": { "value": "{ams.cursor.interactive}" },
-        "font-family": { "value": "{ams.text.font-family}" },
-        "font-size": { "value": "{ams.text.level.4.font-size}" },
-        "font-weight": { "value": "{ams.text.font-weight.bold}" },
+        "font-family": { "value": "{ams.typography.font-family}" },
+        "font-size": { "value": "{ams.typography.heading.4.font-size}" },
```

Comment:
You mentioned this @alimpens  – let’s double check.

This or `{ams.typography.font-size.heading.4}`?